README.md

```diff
@@ -151,16 +152,28 @@ Write something about these
 FOLIO is designed so that different modules can be written in different
 languages with different tools.
 
-#### Java/Vert.x and Node.js
+#### Language Choice
 
-So far we have only written server-side modules in Java, using Vert.x and
-Node.js. Because we use them internally, those two technologies will have
-a prominent place in the FOLIO ecosystem and, initially, it may be easiest
-to get started using them. We provide libraries and utilities that
-help with development (especially with writing standard boiler-plate code and
-scaffolding) but we hope to eventually gain a wide coverage among other
-tools and technologies (e.g. Python, Ruby, etc.). We are counting on an active
-engagement from the community to help out in this area.
+Within FOLIO, server-side modules are primarily written in Java (with some
+in Node.js).  These are used a lot within FOLIO, therefore, we have build
+libraries and utilities to work with them (especially for standard scaffolding
+and boiler-plate code).
+
+Currently, a lot of these modules are written in Java using Vert.x (and built
+with our
+[raml-module-builder](https://github.com/folio-org/raml-module-builder)
+framework).
+
+Some newer modules are written with the 
+[Spring Way](https://docs.google.com/presentation/d/1YgDCBimLTQ1ou-fPhvyKbWpVkec3Goa8lyJJe2hcLHk/edit)
+philosophy.  This uses Spring Boot, a more popular Java framework, as well as
+openapi, making it easier to take advantage of more modern Java features.
+If you are looking to start a new FOLIO module, you will probably want to
+use this methodology and framework as opposed to something RMB/Vert.x based.
+
+We hope to eventually gain a wide coverage among other tools and technologies
```

Comment:
Who is "we"? The tools and technologies should be limited to reduce the maintenance effort, keep CI/CD simple, and avoid learning time for people working on multiple modules.

Reply:
@julianladisch the "we" referred to on 174 was already present (formerly 161) -- just moved to another paragraph.

> but we hope to eventually gain a wide coverage among other
> tools and technologies (e.g. Python, Ruby, etc.). We are counting on an active
> engagement from the community to help out in this area.

We can certainly modify/remove this language, if you think that's best